### PR TITLE
Add Complex radio

### PR DESCRIPTION
**Pull request by TylerCousins**

### CODE REVIEW

File: complexradio/tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "extends";
+    "../tsconfig.json",
+        "compilerOptions";
+    {
+        "outDir";
+        "dist";
+    }
+}
```
Comment (Bas950) [inline complexradio/tsconfig.json:9]: https://docs.premid.app/en/dev/presence#filling-in-the-tsconfigjson-file

File: complexradio/dist/presence.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+var stamp = Math.floor(Date.now());
+presence.on("UpdateData", () => {
+  var info = {
+    largeImageKey: "complexlogo",
+    details: `${presenter} • ${listeners || "Loading statistics"}`,
+    state: `${song || "Loading song"}`,
+    startTimestamp: stamp
+  };
+  presence.setActivity(info);
+});
```
Comment (Bas950) [inline complexradio/dist/presence.js:26]: Presences must be coded in TS and then be compiled to JS now.

File: complexradio/dist/metadata.json [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+      "id": "351991098387988491"
+    },
+    "service": "Complex Radio",
+    "description": {
+      "en": "Complex Radio provides quality music to its listeners, we also always have applications open for new presenters!"
+    },
+    "url": "complexr.pw",
+    "version": "0.1",
+    "logo": "https://i.imgur.com/FZvLxi4.png",
+    "thumbnail": "https://i.imgur.com/FZvLxi4.png",
```
Comment (frozaum) [inline complexradio/dist/metadata.json:13]: Please change the thumbnail to a proper promotional image or a screenshot of the website

Comment (Bas950) [inline complexradio/dist/metadata.json:13]: upload better quality logo
Thumbnail must be an actual thumbnail or a screenshot of the website

File: complexradio/dist/metadata.json [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+    "author": {
+      "name": "Tyler",
+      "id": "351991098387988491"
+    },
+    "service": "Complex Radio",
+    "description": {
+      "en": "Complex Radio provides quality music to its listeners, we also always have applications open for new presenters!"
+    },
+    "url": "complexr.pw",
+    "version": "0.1",
```
Comment (Bas950) [inline complexradio/dist/metadata.json:11]: version 1.0.0

File: complexradio/dist/metadata.json [hunk 5]
```diff
@@ -0,0 +1,25 @@
+{
+    "author": {
+      "name": "Tyler",
+      "id": "351991098387988491"
+    },
+    "service": "Complex Radio",
```
Comment (Bas950) [inline complexradio/dist/metadata.json:6]: Service name and folder name must be the same

File: complexradio/dist/metadata.json (hunk 4 above)
Comment (frozaum) [inline complexradio/dist/metadata.json:11]: Please make sure that the version number follows [semantic versioning standards](https://semver.org/)

**Comment by TylerCousins** [comment]
changed them and the folder name should i make a new put request

**Comment by TylerCousins** [comment]
All fixed

**Comment by TylerCousins** [comment]
Added all new files

**Pull request closed without merging**